### DISCUSSION TITLE
TensorFlow Lite Micro Getting Started | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#147] [Q&A]
# TensorFlow Lite Micro Getting Started | Seeed Studio Wiki

TensorFlow Lite Micro Getting Started

https://wiki.seeedstudio.com/Wio-Terminal-TinyML-TFLM-1/

**Comment by Kasuletrevor** [comment]
The board is not supported by the library. I have tried on several IDE versions and board versions 1.82, 1.8.3. Can you provide a better version?
Thank you.

**Reply by MatthewJeffson** [reply]
Since some wiki documents are little stale, we are encouraging people discover and then maybe make an improvement. Thank you for pointing it out! I have collected it in our list.

**Reply by Kasuletrevor** [reply]
Thank you Matthew for the feedback.
 I have seen the key notice about the commit to look at. It finally worked. Thank you.

**Reply by MatthewJeffson** [reply]
Oh, okay, good to know! <br/> Have a nice day!
